suite-native/biometrics/src/components/BiometricOverlay.tsx

```diff
@@ -53,27 +54,25 @@ export const BiometricOverlay = () => {
     const { applyStyle } = useNativeStyles();
     const { translate } = useTranslate();
     const { isFacialEnabled, isFingerprintEnabled } = useBiometricsSettings();
-    const { isBiometricsAuthenticationCanceled, setIsBiometricsAuthenticationCanceled } =
-        useIsBiometricsAuthenticationCanceled();
+    const { isBiometricsOverlayVisible } = useIsBiometricsOverlayVisible();
+    const { isBiometricsAuthenticationAllowed, requestAuthenticationCheck } = useBiometrics();
 
-    const handleReenable = () => {
-        // Setting this to true lets useBiometrics to ask for biometrics in case it was canceled by user before
-        // https://github.com/trezor/trezor-suite/issues/10647
-        setIsBiometricsAuthenticationCanceled(false);
-    };
     const titleTransKey = getBiometricsTranslationKey({ isFacialEnabled, isFingerprintEnabled });
 
-    return (
+    return isBiometricsOverlayVisible ? (
```

Comment:
Why is this better than previous rendering approach? Currently it's computationally heavier if `!isBiometricsOverlayVisible`, but if it's better in some way, it's okay.

Reply:
Hmm. good thinking. I didnt want to pass params from useBiometrics through props since it only got used in BiometricsOverlay. I'll revisit this